api/datastore/sql/sql.go

```diff
@@ -143,7 +143,7 @@ func (sqlLogsProvider) New(ctx context.Context, u *url.URL) (models.LogStore, er
 func newDS(ctx context.Context, url *url.URL) (*SQLStore, error) {
 	driver := url.Scheme
 
-	log := common.Logger(ctx)
+	log := common.Logger(ctx).WithFields(logrus.Fields{"url": common.MaskPassword(url)})
```

Comment:
Is there a chance that `url` could be be `nil`?

Reply:
@msgodf  bump